src/reporters/json.ts

```diff
@@ -124,6 +127,19 @@ export default class JSONReporter extends BaseReporter {
             });
           });
         }
+        const traces = userTiming.concat(experience, layoutShift);
+        traces.forEach(({ startTime, ts, name, type, duration, endTime }) => {
+          this.writeJSON('journey/timing', journey, {
+            payload: {
```

Comment:
@andrewvc as we cannot use ECS fields for these new type of metrics. Do you have any concern here exposing these metrics via payload or we look for another way to represent these metric data?

Reply:
Coming back to take a second look I'm really glad I caught this comment. These are the sorts of metrics users will want to query with custom dashboards and putting them inside the payload will prevent that.

Beyond user dashboards this prevents us from doing our own aggregations and charts over time it is also a big problem.

Let's take the time to add these fields. The good news is that we can use the new root fields to index these into elasticsearch without having to modify the heartbeat go code. We will have to update the elasticsearch mappings in heartbeat but that is much simpler. You will also have to update the new synthetics mappings in the fleet integration

I am okay with us merging this PR just to get the code in however we should not build any features in the kibana UI based on it until we've moved to values into properly indexed fields

Reply:
Yeah ++ to all your points, Since these features are anyway behind feature flag and heartbeat right now does not even use them, we are better off here and start building the UI once we move them out in the next release. I am also fine with adding them under `root_fields` in the same PR. Do you have any suggestion on the naming?

Reply:
It's tricky, I think let's come up with a name that would make sense to potentially get adopted into ECS extended fields at some point. That said, I do think they likely are too specific for ECS. 

What about `browser.*` as a base for these fields. So, we could use `browser.user_timing.*` for user timing metrics. Perhaps `browser.core_vitals.*` etc. WDYT?

Reply:
++, I like that idea of nesting them under `browser.` Are you fine with them lying under `journey/metrics` type?

Reply:
Moved all fields under `browser.*`